python/hsfs/core/feature_group_api.py

```diff
@@ -52,7 +64,16 @@ def save(self, feature_group_instance):
         )
         return feature_group_object
 
-    def get(self, feature_store_id, name, version, fg_type):
+    def get_smart(
```

Comment:
Just to consider, a more simple name `get` works too since you have replace it with the old `get`.